doc/rst/usingchapel/chplenv.rst

```diff
@@ -693,19 +687,13 @@ CHPL_HWLOC
           ======== ==============================================================
           none     do not build or use jemalloc
           bundled  use the jemalloc distribution bundled with Chapel in third-party
+          system   use the jemalloc found on the system
```

Comment:
The sentence above this table also needs to be updated to reflect the fact there are now 3 options.